src/index.html

```diff
@@ -4,9 +4,345 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a class="header__link" href="#home">
+
+        <img
+          class="header__logo"
+          src="./images/logo.png"
+          alt="Logo"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link is-active">Apple</a>
```

Comment:
1. fix all links (write correct anchors)
2. don't create new BEM block 'is-active', just use modifier
```suggestion
            <a href="#apple" class="nav__link nav__link--is-active">Apple</a>
```